server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -194,9 +185,12 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
         const lumberJackMetric = this.serviceConfiguration.enableLumberTelemetryFramework ?
             Lumberjack.newLumberMetric(LumberEventName.DeliHandler) : undefined;
 
-        if (lumberJackMetric) {
-            lumberJackMetric.setProperties(new Map([[BaseTelemetryProperties.tenantId, this.tenantId],
-            [BaseTelemetryProperties.documentId, this.documentId]]));
+        if (lumberJackMetric)
+        {
```

Comment:
Is there any specific reason to add a newline after "{"? Our convention is to add "{" in the same line as "if".